java/common/src/main/java/org/ray/util/ResourceUtil.java

```diff
@@ -0,0 +1,76 @@
+package org.ray.util;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * The utils of Resource.
+ */
+public class ResourceUtil {
+  public static final String cpuLiteral = "CPU";
+  public static final String gpuLiteral = "GPU";
+
+  public static Map<String, Double> getResourcesMapFromArray(ResourceItem[] resourceArray) {
```

Comment:
Document these methods in this class. (Old Java code is lack of documents, let's start adding documents for new code :)